apis/v1beta1/validation/httproute_test.go

```diff
@@ -1140,3 +1142,45 @@ func TestValidateHTTPRouteTypeMatchesField(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateHTTPTimeouts(t *testing.T) {
+	tests := []struct {
+		name     string
+		rules    []gatewayv1b1.HTTPRouteRule
+		errCount int
+	}{{
+		name:     "valid httpRoute Rules timeouts",
+		errCount: 0,
+		rules: []gatewayv1b1.HTTPRouteRule{{
+			Timeouts: &gatewayv1b1.HTTPRouteTimeouts{
+				Request: &metav1.Duration{
+					Duration: time.Duration(time.Millisecond * 1),
+				},
+			},
+		},
+		}}, {
+		name:     "invalid httpRoute Rules backendRequest timeout cannot be longer than request timeout",
+		errCount: 1,
+		rules: []gatewayv1b1.HTTPRouteRule{{
+			Timeouts: &gatewayv1b1.HTTPRouteTimeouts{
+				Request: &metav1.Duration{
+					Duration: time.Duration(time.Millisecond * 1),
+				},
+				BackendRequest: &metav1.Duration{
+					Duration: time.Duration(time.Millisecond * 2),
+				},
+			},
+		},
+		}},
```

Comment:
Maybe add a couple more tests?
1. 0 -> valid
2. 0.5 ms -> invalid

Reply:
addressed in 4056411a2ddccafe6f9eb40bb9b14fd685e1859e